include/mbgl/ios/MGLMapView_IBAdditions.h

```diff
@@ -0,0 +1,25 @@
+/*
+    This category header is named with an underscore instead of a plus sign
+    because Interface Builder sorts inspectables in ASCII alphabetical order by
+    the names of the headers that declare them, and we want the core MGLMapView
```

Comment:
Yeah. :smile: Alternatively, I could redeclare the core properties here with the `IBInspectable` keyword and rename this file to match the + convention.